.github/workflows/weekly-resources-clean.yml

```diff
@@ -17,10 +17,29 @@ on:
   schedule:
     - cron: "0 0 * * SAT" # Run every saturday at midnight
   workflow_dispatch:
+    inputs:
+      daystokeep:
+        description: "days to keep a resource"
+        required: true
```

Comment:
IMO, we don't need this to be required, set this as a default value by using ```github.event.inputs.daystokeep || 5``` would simpler though

Reply:
Sure. I think it's not a big deal since this is for manual runs, which shouldn't normally happen anyway.

Reply:
Feels a bit weird to have inputs without one of the fields required. At least to me.